### ISSUE TITLE
Rails 5

### ISSUE THREAD

**Issue by rhymes** [brandonweiss/rack-minitest#5]
I'm not sure it works with Rails 5...

Also Rails 5 introduced ActionDispatch::IntegrationTest class that supports posting as JSON http://api.rubyonrails.org/classes/ActionDispatch/IntegrationTest.html

**Comment by brandonweiss** [comment]
I haven't upgraded any apps to Rails 5 yet, but I'll take a look at this as soon as I do 😄

**Comment by rhymes** [comment]
@brandonweiss yeah, me neither, I just created a branch of one of my Rails 4.2 apps to see how hard it was to upgrade and noticed some issues :)
